<launch>
	<param name="tf_prefix" value="robot_2"/>
  <node pkg="tcp_interface" type="tcp_interface" name="tcp3" output="screen">
	<param name="TCP_server_port" value="9002"/>
    <param name="UDP_server_port" value="9102"/>
    <param name="network_interface" value="wlan0"/>
    <param name="UDP_team_min_port" value="9100"/>
    <param name="UDP_team_max_port" value="9110"/>
    <param name="config_file" value="$(find tcp_interface)/config/UDPpeers.cfg"/>
    

  
  </node>

</launch>
